src/resolve-template.js

```diff
@@ -0,0 +1,90 @@
+var path = require("path");
+var fileSystem = require('fs');
+var readdir = require('recursive-readdir');
+var ContextElementDependency = require('webpack/lib/dependencies/ContextElementDependency');
+var assign = Object.assign || require('object.assign');
+
+var cheerio = require('cheerio');
+
+/**
+ * Generates key-value pairs or <require from="paths"> and their corresponding files.
+ */
```

Comment:
I'm currently working on even more extensive scraping of requires, such as `view-model="file"` and `view="file.html"`. @EisenbergEffect Are there any more ways of importing files/modules dynamically from the view?